src/docs/src/ddocs/index.rst

```diff
@@ -26,6 +26,7 @@ replications.
 
     ddocs
     views/index
+    mango
```

Comment:
I'm not convinced this documentation should live under the "design documents" documentation. Mango does use design documents under the hood but it's an implementation detail - users shouldn't be manually editing / interpreting them. 

Additionally, the `_find`, `_explain` and `_index` endpoints are distinct from design document endpoints, unlike `_view`/`_search`/`_nouveau` which sit underneath the `<db>/_design/<ddoc_name>` URL root.

Reply:
Perhaps this is as simple as breaking out a "querying data" topic from the "design documents" guide.

Reply:
I won't say you're wrong, but that sounds like scope creep, and as an external contributor, I would quite like to get this PR finished.